sgx-tools/dcap_linux.go

```diff
@@ -0,0 +1,514 @@
+package main // import "github.com/inclavare-containers/sgx-tools"
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-restruct/restruct"
+	"github.com/golang/protobuf/proto"
+	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	pb "github.com/inclavare-containers/sgx-tools/proto"
+	"github.com/sirupsen/logrus"
+	"net"
+	"strings"
+)
+
+const (
+	aesmdSocket                  = "/var/run/aesmd/aesm.socket"
+	attestationKeyIdentityLength = 256
+	nonceLength                  = 16
+	qeReportInfoLength           = 960
+	// In millisecond
+	timeOut          = 15000
```

Comment:
This is the global const for aesmd transition timeout. We need a global name: `aesmdTimeout`.